README.md

```diff
@@ -23,13 +23,70 @@ $ npx @commercetools-frontend/create-mc-app my-new-custom-application-project --
 
 ## Developing application-kit packages
 
-Install the dependencies (uses yarn workspaces):
+### Initial Installation
 
-```bash
-$ yarn
-```
+1. Clone this repo
+
+   ```bash
+   $ git clone https://github.com/commercetools/merchant-center-application-kit.git
+   ```
+
+2. Add necessary environment variables
+
+   > In the vscode file tree or the terminal, navigate to `merchant_center_application_kit/playground`, open `.env.local.template`, and add these values:
```

Comment:
This file shouldn't be modified––you'll want to duplicate it and rename to `.env.local` instead. No need to specify the method you'll use to do it, either; I'm sure there's an Emacs user in the org somewhere 😉

Reply:
good point, changed in most recent commit